_data/chains/eip155-314.json

```diff
@@ -31,8 +35,8 @@
       "standard": "EIP3091"
     },
     {
-      "name": "Filmine",
-      "url": "https://explorer.filmine.io",
```

Comment:
why do we remove this one and not just add the other one?

Reply:
They are rebranding